SIGS_AND_WGS.md

```diff
@@ -0,0 +1,25 @@
+# SIGs and WGs
```

Comment:
I'm ambivalent about separate files, but if we do separate I see no need to separate sub-projects from SIGs. 
I agree with your subsequent points. @dhiller, I can help summarise some key paras from the k8s docs if you'd like

Reply:
I agree that usage of subprojects is missing here, I'll update the PR wrt that. I am not sure whether subprojects should be inside the filename though. I'd be against splitting into separate files, since having sections does a good enough job - in contrast to having a file WG.md where only one sentence is contained. WDYT?

To the latter point - WDYM? Isn't referencing the Kubernetes model as seen below enough? Why should we repeat the definitions?

Reply:
^^ @EdDev

Reply:
^^ @EdDev

Reply:
I can't speak for Ed but imho something along the lines of :
"We follow the definition of a SIG: A SIG is comprised of members from multiple companies and organizations, with a common purpose of advancing the project with respect to a specific topic, such as Networking or Documentation. Our goal is to enable a distributed decision structure and code ownership, as well as providing focused forums for getting work done, making decisions, and onboarding new contributors.
For more information, see the k8s docs"
Or similar.

The reason being is that it gives the casual reader the context for what this is all about without having to follow links, which means they can understand the page on its own merits, and the links are there for them to follow fro greater/more specific understanding.
I don't expect the k8s descriptions are likely to change all that often so I don't really see an issue with maintenance.

Reply:
>  am not sure whether subprojects should be inside the filename though. I'd be against splitting into separate files, since having sections does a good enough job - in contrast to having a file WG.md where only one sentence is contained. WDYT?

SIGs and WG are IMO separated topics and better appear in different files.
I agree that subprojects fit the SIG file as they are part of it.
The current name is very odd IMO, trying to name two different things in one name.

Reply:
^^ @aburdenthehand @EdDev WDYT?

@aburdenthehand what you said makes sense - a short collection of what's this about makes sense

Reply:
@dhiller We currently have a Governance.md file in the repo: it is largely, but not exclusively!, concerned with governance of project maintainers. Yeah, maybe that would be a better place for this content (summary of SIGs and WGs). 
We can also update the Readme link to point to their particular sections in the governance doc.